Vagrantfile

```diff
@@ -1,10 +1,51 @@
 $VM_MEMORY = (ENV['VM_MEMORY'] || 8912)
 $VM_CPUS = (ENV['VM_CPUS'] || 2)
+$GO_VERSION = (ENV['GO_VERSION'] || "1.22.0")
+
+$go_install = <<-'SCRIPT'
+# Install golang
+GO_VERSION=$1
+curl -O https://storage.googleapis.com/golang/go$GO_VERSION.linux-amd64.tar.gz && \
+    rm -rf /usr/local/go \
```

Comment:
I don't think we  want to delete existing go on users. This is on the host?

Could we check if go version is correct first and then give a readable warning with instructions on how to do this? And/or prompt to replace?